bot/library/tezos.js

```diff
@@ -73,9 +73,15 @@ module.exports = class Tezos {
    * @param address tezos address for the account
    */
   async tokenBalance(tokenContract, address) {
-    const key = TezosMessageUtils.encodeBigMapKey(
+    let key = TezosMessageUtils.encodeBigMapKey(
```

Comment:
wrap this logic in a single if/else block for `!== 31 // txbtc` and `=== 31`

Reply:
It is already doing a conditional check for the map id, I kept it outside because I needed a default value for the key and instead of doing if else I kept the else value as the default.